### ISSUE TITLE
macOS Version Issue

### ISSUE THREAD

**Issue by 57194** [onionshare/onionshare#1914]
Downloading and installing `OnionShare-2.6.2.dmg` from https://github.com/onionshare/onionshare/releases/tag/v2.6.2 installs 2.6.1, so OnionShare is saying it's out-of-date and needs to be updated…

**Comment by nicfab** [comment]
Indeed, although I downloaded the latest version (2.6.2) from Settings, the version results were 2.6.1.

**Comment by SaptakS** [comment]
Thanks for opening this issue. I will take a look at this to see if there is some versioning issue somewhere.
